knowledge_base/HABITAT.md

```diff
@@ -1,4 +1,61 @@
 # Working with [`habitat-lab`](https://github.com/facebookresearch/habitat-lab)
 
-We are currently working on integrating habitat-lab environments into torchrl.
-Stay tuned for more info on this.
+## Setting up your environment for habitat and torchrl
+
+### Installing habitat-lab from pip
+
+Instructions can be found on [habitat github repo](https://github.com/facebookresearch/habitat-lab).
+
+  1. **Preparing conda env**
+
+     Assuming you have [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) installed, let's prepare a conda env:
+     ```bash
+     export MY_TEST_ENV=habitat_test
+     conda deactivate
+     conda env remove -n $MY_TEST_ENV -y
+     conda create -n $MY_TEST_ENV python=3.7 cmake=3.14.0 -y
+     conda activate $MY_TEST_ENV
+     ```
+
+  2. **conda install habitat-sim**
+     To install habitat-sim with bullet physics and in headless mode (usually necessary to run habitat on a cluster)
+     ```bash
+     conda install habitat-sim withbullet headless -c conda-forge -c aihabitat-nightly -y
+     pip install git+https://github.com/facebookresearch/habitat-lab.git#subdirectory=habitat-lab
+     
+     # This is to reduce verbosity
+     export MAGNUM_LOG=quiet && export HABITAT_SIM_LOG=quiet
+     ```
+     If you don't want to install it in headless mode, simply remove the `headless` package from the `conda install` command.
+
+     See Habitat-Sim's [installation instructions](https://github.com/facebookresearch/habitat-sim#installation) for more details.
+
+  3. **Install habitat-baselines**.
```

Comment:
I wonder if habitat_baselines are needed for torchrl. It seems the main use case will be to create and wrap environments (which is what habitat-lab does) without the need to use the habitat-baselines trainers.

Reply:
Yes we probably don't need that, I can remove it